docs/clustering/logical-replication-setup.rst

```diff
@@ -0,0 +1,148 @@
+statements discussed below can be executed via the corresponding Admin UI.
+
+As the subscriber pulls changes from the publisher, the publisher needs to accept
+incoming network connections from the subscriber. In our example setup, this is
+automatically given because both containers are running on the same host.
+For reaching each other, they will use the special ``host.docker.internal`` address.
+
+Setting up a publication
+========================
+
+.. NOTE::
+
+  All SQL statements in this section are executed on ``cluster-publishing``.
+
+Before setting up the replication, we create a simple table that is going to be
+the subject of replication. A table with the same name must not exist yet on
+``cluster-subscribing``.
+
+.. code-block:: sql
+
+  CREATE TABLE doc.temperature_data (
+    ts TIMESTAMP NOT NULL,
+    temperature FLOAT NOT NULL
+  );
+
+Next, a publication is created. The publication marks our table as being
+available for replication, but otherwise does not imply any activity yet.
+
+.. code-block:: sql
+
+  CREATE PUBLICATION temperature_publication FOR TABLE doc.temperature_data;
+
+To verify the publication was created successfully, we query the
+``pg_publication`` system table. It should contain one row with the publication
+just added.
+
+.. code-block:: sql
+
+  SELECT *
+  FROM pg_publication;
+
+
+With this, we are already all set on the publication side.
+
+Setting up a subscription
+=========================
+
+.. NOTE::
+
+  All SQL statements in this section are executed on ``cluster-subscribing``.
+
+A subscription needs connection information to the publishing cluster as
+well as the name of the previously created publication to subscribe to.
+
+Using the ``mode`` parameter, the connection can be established via either the
+transport protocol or the PostgreSQL protocol. By setting the parameter to
+``pg_tunnel``, we use the PostgreSQL protocol.
+
+For a full overview of available parameters, please see the CREATE SUBSCRIPTION
+reference.
+
+.. code-block:: sql
+
+  CREATE SUBSCRIPTION temperature_subscription
+  CONNECTION 'crate://host.docker.internal:5433?user=crate&mode=pg_tunnel'
+  PUBLICATION temperature_publication;
+
+.. NOTE::
+
+  The ``password`` parameter is omitted, as local connections using the ``crate``
+  user don't require one. If you are connecting remotely, provide ``user`` and
+  ``password`` of a user with ``DQL`` privileges on published tables.
+
+After a few seconds, the table ``doc.temperature_data`` should appear on
+``cluster-subscribing``. At this point, it is still empty as we didn't insert
+any data yet.
+
+To verify the operational status of the subscription, the system tables
+pg_subscription and pg_subscription_rel can be queried. The below query returns
+the name of the subscription (``subname``), its state (``srsubstate``), as well
+as any potential error message (``srsubstate_reason``).
+
+.. code-block:: sql
+
+  SELECT subname, srsubstate, srsubstate_reason
```

Comment:
r.srrelid::text gets a table name - might be useful to see the table name as status is granular per table.